src/rust/engine/dep_inference/src/python/mod.rs

```diff
@@ -149,48 +153,68 @@ impl ImportCollector<'_> {
     false
   }
 
+  fn normalize_import_node(node: tree_sitter::Node) -> Option<tree_sitter::Node> {
+    match node.kind_id() {
+      KindID::ALIASED_IMPORT => node.named_child(0),
+      KindID::WILDCARD_IMPORT => None,
+      KindID::ERROR => None,
+      _ => Some(node),
+    }
+  }
+
+  /// Various different styles of references to modules.
+  ///
+  /// NB. this diverges slightly from the core tree-sitter grammar by calling base module reference
+  /// (the part that always exists) "module_name", with "name" only used to refer to the extra bit
+  /// in `from ... import ...`:
+  ///
+  /// ```python
+  /// import $base_name
+  /// from $base_name import * # (the * node is passed as imported_name too)
+  /// from $base_name import $imported_name
+  /// "$base_name"
```

Comment:
Can you put this under the vanilla import and then add a comment saying it's a string import (or just remove it altogether).

It took be waay too long to realize this was string imports 😂